src/com/dabomstew/pkrandom/constants/Moves.java

```diff
@@ -0,0 +1,757 @@
+package com.dabomstew.pkrandom.constants;
+
+/*----------------------------------------------------------------------------*/
+/*--  Moves.java - defines an index number constant for every Move through  --*/
+/*--  the Let's Go series.                                                  --*/
```

Comment:
This actually isn't accurate, as it's missing Zippy Zap (729) to Double Iron Bash (742). The way I see it, you can do three things here:
- Just change this (and the PR description) to say "Ultra Sun/Ultra Moon" instead
- Add Zippy Zap to Double Iron Bash to make this comment accurate
- Go all the way and add all the moves, similar to how Species.java and Abilities.java go up to the Crown Tundra even though the randomizer doesn't support it yet

The first one is the easiest but I'd be fine with any of them.

Reply:
For the sake of consistency with how we handled species/abilities, I'd like to see the third option here - might as well future proof it.

Reply:
Done.